package.json

```diff
@@ -14,7 +14,7 @@
     "test-setup:happ": "nix-shell --command 'yarn fetch:dna && yarn build:happ' && exit",
     "test-setup:holo": "yarn run clear:hc-holo && nix-shell --command 'yarn hc-holo:gen-agent && yarn hc-holo:run && yarn run:test-chaperone-hcc' && exit",
     "test:integration-hc": "yarn run clear:hc && yarn run clear:lair && yarn build:holochain && nix-shell --command 'VUE_APP_CONTEXT='holochain' VUE_APP_WEB_LOGS='false' VUE_APP_TEST_PORT='8888' node scripts/test.js --config=./jest.hcintegration.config.js --watch --runInBand --detectOpenHandles --testTimeout=95000000' && exit",
-    "test:integration-holo": "yarn test-setup:holo && yarn build:holo-hcc && VUE_APP_CONTEXT='holo-host' VUE_APP_WEB_LOGS='false' VUE_APP_CHAPERONE_SERVER_URL='http://localhost:24273/' node scripts/test.js --config=./jest.holointegration.config.js --watch --runInBand --detectOpenHandles --testTimeout=95000000",
```

Comment:
They weren't doing anything because they were placed on the node command that spawns the puppeteer tester, not the command that builds the website. `yarn build:holo-hcc` (which is included in this line) does have those variables set where it actually matters.